packages/radio-group/src/Radio/Radio.styles.ts

```diff
@@ -14,13 +14,26 @@ export const inputDisplayWrapperClassName =
   createUniqueClassName('radio-group');
 export const inputClassName = createUniqueClassName('radio-group');
 
-export const containerMargin = css`
+export const containerStyle = css`
+  display: grid;
+  grid-template-areas: 'label label' '. description';
+
   & + & {
```

Comment:
You could also remove this bit of code and replace with flex and gap. This way, teams can use flex-direction.

Reply:
Ah good call!!